packages/lesswrong/components/review/ReviewVotingPage.tsx

```diff
@@ -16,22 +16,23 @@ import CachedIcon from '@material-ui/icons/Cached';
 import { Link } from '../../lib/reactRouterWrapper';
 import { AnalyticsContext, useTracking } from '../../lib/analyticsEvents'
 import seedrandom from '../../lib/seedrandom';
-import { currentUserCanVote, getReviewPhase, REVIEW_NAME_IN_SITU, REVIEW_NAME_TITLE, REVIEW_YEAR } from '../../lib/reviewUtils';
+import { currentUserCanVote, getReviewPhase, REVIEW_NAME_IN_SITU, REVIEW_YEAR } from '../../lib/reviewUtils';
 import { annualReviewAnnouncementPostPathSetting, annualReviewStart } from '../../lib/publicSettings';
 import moment from 'moment';
 import { forumTypeSetting } from '../../lib/instanceSettings';
+import Select from '@material-ui/core/Select';
+import MenuItem from '@material-ui/core/MenuItem';
+import Card from '@material-ui/core/Card';
 
 const isEAForum = forumTypeSetting.get() === 'EAForum'
 
-const VOTING_VIEW = "reviewVoting" // unfortunately this can't just inhereit from REVIEW_YEAR. It needs to exactly match a view-type so that the type-check of the view can pass.
```

Comment:
The original purpose of these variables was to make it easier to keep track of how we created new views each year (i.e. reviews2018, reviews2019), which is no longer necessary.